### PR TITLE
feat(log): AWSX-684 Adding extraction of VPC FlowLog arn

### PR DESCRIPTION
**Pull request by ViBiOh**
### What does this PR do?

Extract the ARN of the VPC FlowLog being processed.

### Motivation

Enrich logs with tags of the VPC FlowLog on Datadog

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [ ] This PR passes the integration tests (ask a Datadog member to run the tests)
- [ ] This PR passes the unit tests 
- [ ] This PR passes the installation tests (ask a Datadog member to run the tests)

### CODE REVIEW

File: aws/logs_monitoring/steps/handlers/s3_handler.py [hunk 1]
```diff
@@ -219,6 +216,25 @@ def parse_service_arn(source, key, bucket, context):
...
+        if len(namesplit) >= 4:
+            accountID = namesplit[1]
+            region = namesplit[3]
+
+            filesplit = namesplit[7].split("_")
+
+            if len(filesplit) >= 4:
+                identifier = filesplit[3]
+
+                return "arn:{}:ec2:{}:{}:vpc-flow-log/{}".format(
```
Comment (ge0Aja) [inline aws/logs_monitoring/steps/handlers/s3_handler.py:231]: Do we want to use this ARN directly on the logs backend side ?

**Review by ge0Aja — APPROVED** [review]
LGTM

**Review by ge0Aja — APPROVED** [review]
LGTM

File: aws/logs_monitoring/steps/handlers/s3_handler.py (hunk 1 above)
Comment (ViBiOh) [reply aws/logs_monitoring/steps/handlers/s3_handler.py:231]: Yes, I have a [draft pull-request for this](https://github.com/DataDog/logs-backend/pull/66877/files#diff-9c896f5dc6eec1b4a0af62a7f4750730e0bd0fb4ae5bb8ad3b7942f7e2f9351dR21-R25), but we need to discuss some implementation details.

**Pull request closed without merging**